src/infrastructure/sequelize/models/favori-offre-emploi.sql-model.ts

```diff
@@ -64,4 +64,10 @@ export class FavoriOffreEmploiSqlModel extends Model {
 
   @Column({ field: 'date_creation', type: DataType.DATE })
   dateCreation: Date | null
+
+  @Column({ field: 'origine_nom', type: DataType.STRING })
+  origineNom: string | null
+
+  @Column({ field: 'origine_logo', type: DataType.STRING })
+  origineLogo: string | null
```

Comment:
`origineLogoSrc` ou `origineLogoUrl` ? C'est pas clair que c'est une url.

Reply:
ok why not